template.tpl

```diff
@@ -2023,6 +2085,23 @@ switch (data.eventType) {
       return fail('Unable to track any valid Enhanced Ecommerce event.');
     }
     break;
+    
+  case 'enableButtonClickTracking':
+    commandName = data.eventType;
+    
+    // `data.buttonClickTrackingFilterChoice` will always be set,
+    // being "allowlist" by default
+    const choice = data.buttonClickTrackingFilterChoice;
+    const filters = data.buttonClickTrackingFilters
```

Comment:
Since this can be configured as a variable we might want to check if it's an array already unless we want to mandate strings only.